### PR TITLE
Submit HW01 (wanitchaya)

### PR DESCRIPTION
**Pull request by wanitchayap**
I changed the README.md by Fork and clone the repository from the GitHub into my computer first. The cloning confused me when I tried to commit and push my updated files in the first place because I cloned using the GitKraken. However, in the end, I committed and pushed by following the process in the 'homework workflow' manually without using the GitKraken and with helps from Chi Chian. After I have the Git linked to the RStudio, I put my pictures into the repository folder in my computer. I used the Rmarkdown cheatsheet to write my biography. I kept checking the HTML result of my README.md to make sure the visual result was what I intended it to be. Finally, I committed and pushed all the files on the RStudio and made this pull request on the GitHub.

### CODE REVIEW

**Comment by chichian93** [comment]
Evaluation HW-01
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |       x    |              |            |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:
You have provided a very comprehensive self-introduction and experimented with more than 4 aspects of the Markdown syntax in this assignment. In fact, you explored functions such as quote, which was beyond what was stipulated. 

I noticed that the way you insert hyperlink is different from the way I did it, and the method you used is neater in my opinion. That is something useful which I have learned from your work! =D

P.S. Let's go for spicy food some day!! The only legit ones I tried since coming to Chicago were at Chinatown and the spicy-pork-rice-bowl from the food truck in school...

File: README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
 # hw01
 Repository for homework 01
```
Comment (lesliesmei) [inline README.md:2]: Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |    x       |              |            |

Remarks:

* This was a nicely formatted, clean and readable biography.
* I especially liked the placement and choice of photos.
* You definitely went above and beyond demonstrating different markdown styles!
* If I had to criticize, I would say that you should delete the 'image:' lines and let the images speak for themselves.

Really nice work!!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,60 @@
...
+  * Quote
+        
+> Grown-ups never understand anything by themselves, and it is tiresome for children to be always and forever explaining things to them ― *Antoine de Saint-Exupéry, The Little Prince*
+
+### Mint Online
+  * [Facebook][1]
+  * [LinkedIn][2]
+
+[1]: https://www.facebook.com/mintwanitchaya
+[2]: https://www.linkedin.com/in/wanitchaya-poonpatanapricha-631b20131/
```
Comment (lpstntw) [inline README.md:60]: Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            | X         |              |            |
| **Coding strategy**         | X         |              |            |
| **Presentation**            | X         |              |            |
| **Achievement, creativity** | X         |              |            |
| **Ease of access**          | X         |              |            |

Remarks:

* I love the way you format the whole markdown. It's clear-cut and lucid.
* I like that you put the further information link at the end of introduction.

**Review by lpstntw — COMMENTED** [review]
Great Job!!!

**Comment by mayank-kgp** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   x        |              |            |
| **Coding strategy**         |       x    |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* You have added multiple images, used markdown very well. Page design is very good. It's perfect. Just a quick note - You don't need to commit all files such as .gitignore, Rproj or .DS Store files. Just commit those files which are necessary.

**Pull request closed without merging**